lib/upgrader.js

```diff
@@ -68,6 +132,33 @@ module.exports = ({ argv }) => {
     additional code upgrade concerns requiring manual attention.
   `);
 
+  function processLibModules() {
+    // Only touch directories, don't mess about with regular files in lib/modules (WTF)
```

Comment:
What's the "WTF"? We should avoid vague reaction comments in favor of clear instructions.